README.md

```diff
@@ -1,8 +1,127 @@
-# MAIN README
+# PPRZLINK
 
-PPRZLINK 
-========
+## Overview
 
-PPRZLINK is a messages toolkit (message definition, code generators, libraries) to be used with [Paparazzi UAV System] (https://paparazziuav.org) and compatible systems
+PPRZLINK is a messages toolkit (message definition, code generators, libraries) to be used with [Paparazzi UAV System] (https://paparazziuav.org) and compatible systems. One tool that uses PPRZLINK is the [Flying Robot Commander](https://github.com/paparazzi/flyingrobotcommander), a web based, RESTful application for controlling multiple aircraft.
 
+## Libraries
 
+PPRZLINK libraries are available for the following programming languages: 
+
+- [C](https://github.com/hooperfly/pprzlink/tree/master/lib/v1.0/C)
+- [ocaml](https://github.com/hooperfly/pprzlink/tree/master/lib/v1.0/ocaml)
+- [python](https://github.com/hooperfly/pprzlink/tree/master/lib/v1.0/python)
+
+## Python Library
+
+The [python](https://www.python.org/) PPRZLINK ivy interface is defined in [ivy_msg_interface.py](https://github.com/paparazzi/pprzlink/blob/master/lib/v1.0/python/ivy_msg_interface.py). There is also a serial version of the interface in [serial_msg_interface.py](https://github.com/paparazzi/pprzlink/blob/master/lib/v1.0/python/serial_msg_interface.py).
+
+### Ivy Message Call Sequence
+
+Add Libraries to the Search Path
+
+    # if PAPARAZZI_SRC not set, then assume the tree containing this file is a reasonable substitute
+    PPRZ_SRC = getenv("PAPARAZZI_SRC", path.normpath(path.join(path.dirname(path.abspath(__file__)), '~/paparazzi/')))
+
+    sys.path.append(PPRZ_SRC + "/sw/lib/python")
+    sys.path.append(PPRZ_SRC + "/sw/ext/pprzlink/lib/v1.0/python")
+
+    from ivy_msg_interface  import IvyMessagesInterface
```

Comment:
sorry, clicked on the wrong line
`sys.path.append(PPRZ_SRC + "/sw/lib/python")`
is not needed